index.html

```diff
@@ -40,5 +40,205 @@ <h1>I'm Benjamin Glad to see you!</h1>
+          <li>javascript</li>
+        </ul>
+      </div>
+
+      <button class="project-link" type="submit">See Project</button>
+    </section>
+
+  
+    <section class="middle-section-container">
+      <div class="image-area">
+        <img src="photo/logos/Snapshoot-Portfolio3.png" alt="">
+      </div>
+
+        <div class="hero">
+          <div class="tonic">
+            <h1>Tonic</h1>
+              <div class="canopy-wrapper">
+                <p>CANOPY</p>
+                  <ul>
+                   <li>Back End Dev</li>
+                     <li>2015</li>
+                  </ul>
+              </div>
+          </div>
+
+            <div class="text-area">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+            </div>
+
+          <div class="middle-section-nav-bars">
+            <ul class="technology">
+             <li>html</li>
+             <li>css</li>
+             <li>javascript</li>
+           </ul>
+          </div>
+
+          <button class="project-link" type="submit">See Project</button>
+        </div>
+
+    </section>
+    <section class="middle-section-container2">
+      <div class="image-area">
+        <img src="photo/logos/Snapshoot-Portfolio4.png" alt="">
+      </div>
+
+      <div class="hero">
+        <div class="tonic2">
+            <h1>Multi-Post Stories</h1>
+            <div class="canopy-wrapper">
+              <p>CANOPY</p>
+              <ul>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+        </div>
+
+          <div class="text-area">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+          </div>
+
+          <div class="middle-section-nav-bars">
+            <ul class="technology">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+
+          <button class="project-link" type="submit">See Project</button>
+      </div>
+    </section>
+
+    <section class="about-myself">
+      <h1 class="header">
+        About Myself
+      </h1>
+
+      <div class="header-text">
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </div>
+      
+      <button class="about-myself-link" type="submit">LET'S CONNECT</button>
+
+      <div class="about-myself-icons">
+        <img src="photo/logos/github.jpg" alt="">
+        <img src="photo/navs/Linkedin-icon.jpg" alt="">
+        <img src="photo/logos/angellist.jpg" alt="">
+        <img src="photo/logos/twitter.jpg" alt="">
+        <img src="photo/logos/medium.jpg" alt="">
+      </div>
+      
+      <button class="resume" type="submit">Get my resume</button> 
```

Comment:
- [ ] [OPTIONAL] The button has got vertical padding in the design. Please make it look like the design as much as possible.